cmd/difference.go

```diff
@@ -274,7 +274,7 @@ func differenceInternal(ctx context.Context, sourceClnt, targetClnt Client, sour
 					firstContent:  srcCtnt,
 					secondContent: tgtCtnt,
 				}
-			} else if activeActiveModTimeUpdated(srcCtnt, tgtCtnt) {
+			} else if activeActiveModTimeUpdated(srcCtnt, tgtCtnt, isActiveActive) {
```

Comment:
Can we call `activeActiveModTimeUpdated` only if `isActiveActive = true`?

Reply:
The problem with that is we need modification time check for non-active-active setups too.